src/unix/linux-core.c

```diff
@@ -718,7 +718,7 @@ static int read_models(unsigned int numcpus, uv_cpu_info_t* ci) {
 #if defined(__arm__)
       /* Fallback for pre-3.8 kernels. */
       static const char model_marker[] = "Processor\t: ";
-#else	/* defined(__mips__) */
+#else /* defined(__mips__) */
```

Comment:
You're right, but the change is not related to this PR and adds noise to the diff.